Update Windows compatibility to WSL in tutorials (#2)
cyberbotics/ros2_documentation: source/Tutorials/Advanced/Simulators/Webots.rst

It is recommended to understand basic ROS principles covered in the beginner :doc:`../../../Tutorials`.
In particular, :doc:`../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim`, :doc:`../../Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics`, :doc:`../../Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace`, :doc:`../../Beginner-Client-Libraries/Creating-Your-First-ROS2-Package` and :doc:`../../Intermediate/Launch/Creating-Launch-Files` are useful prerequisites.
Finally, you will need to install ``webots_ros2_driver`` from a terminal with the following commands. 

> On Windows, a WSL environment must be configured and used to run all the console and ROS commands contained in this tutorial. 

Reviewer: ```suggestion
On Windows, a WSL (Windows Subsystem for Linux) environment must be configured and used to run all the Linux and ROS commands contained in this tutorial. 
```

It is recommended to understand basic ROS principles covered in the beginner :doc:`../../../Tutorials`.
In particular, :doc:`../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim`, :doc:`../../Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics`, :doc:`../../Beginner-Client-Libraries/Creating-A-Workspace/Creating-A-Workspace`, :doc:`../../Beginner-Client-Libraries/Creating-Your-First-ROS2-Package` and :doc:`../../Intermediate/Launch/Creating-Launch-Files` are useful prerequisites.
Finally, you will need to install ``webots_ros2_driver`` from a terminal with the following commands. 
On Windows, a WSL environment should be configured and used to execute commands. 
This `page <https://github.com/cyberbotics/webots_ros2/wiki/Build-and-Install#windows>`_ explains how to setup such installation. 

.. code-block:: console

>         # On Windows, execute these in the WSL environment

Reviewer: I find this sentence (repeated several times) is a bit confusing as it seems we should execute these lines this ONLY on Windows, not on Linux. I would remove these sentences and add a sentence earlier that explain that all the Ubuntu and ROS commands should be entered in WSL for Windows users.

In your case, you need to run a single instance of this node, because you have a single robot in the simulation.
But if you had more robots in the simulation, you would have to run one instance of this node per robot.
``WEBOTS_CONTROLLER_URL`` is used to define the name of the robot the driver should connect to.

> On Windows, the communication between Webots and the controllers, running in WSL, is possibly via TCP connection.

Reviewer: ```suggestion
On Windows, the communication between Webots (running in Windows) and the controllers (running in WSL) goes through a TCP connection.
```
